www/assessment/reset-response.php

```diff
@@ -0,0 +1,173 @@
+        </div>
+    </div>
+
+    <!--  Question 1  -->
+    <div class="section pad-sml">
+        <h3">Multiple Choice</h3>
+        <hr>
+        <span class="learnosity-item" data-reference="demo_reset_response_act_1_no_edit"></span>
+        <div class="button-wrapper">
+            <button
+                type="button"
+                class="resetResponseButton"
+                data-question-button-index="0"
+            >
+                <span>Reset Response</span>
+            </button>
+        </div>
+        <div class="button-note">
+            <br>
+            <p>NOTE: The Reset Response button is a custom button that calls the <a href="https://reference.learnosity.com/questions-api/methods#questionsApp-ResetResponse"><code>resetResponse</code></a> public method.</p>
+        </div>
+    </div>
+
+    <!--  Question 2  -->
+    <div class="section pad-sml">
+        <h3">Choice Matrix</h3>
+        <hr>
+        <span class="learnosity-item" data-reference="demo_reset_response_act_2_no_edit"></span>
+        <div class="button-wrapper">
+            <button
+                type="button"
+                class="resetResponseButton"
+                data-question-button-index="1"
+            >
+                <span>Reset Response</span>
+            </button>
+        </div>
+        <div class="button-note">
+            <br>
+            <p>NOTE: The Reset Response button is a custom button that calls the <a href="https://reference.learnosity.com/questions-api/methods#questionsApp-ResetResponse"><code>resetResponse</code></a> public method.</p>
+        </div>
+    </div>
+
+    <!--  Question 3  -->
+    <div class="section pad-sml">
+        <h3">Cloze Dropdown</h3>
+        <hr>
+        <span class="learnosity-item" data-reference="demo_reset_response_act_3_no_edit"></span>
+        <div class="button-wrapper">
+            <button
+                type="button"
+                class="resetResponseButton"
+                data-question-button-index="2"
+            >
+                <span>Reset Response</span>
+            </button>
+        </div>
+        <div class="button-note">
+            <br>
+            <p>NOTE: The Reset Response button is a custom button that calls the <a href="https://reference.learnosity.com/questions-api/methods#questionsApp-ResetResponse"><code>resetResponse</code></a> public method.</p>
+        </div>
+    </div>
+
+    <!--  Load items API  -->
+    <script src="<?php echo $url_items; ?>"></script>
+    <script>
+
+        // Function that calls the Learnosity resetResponse public method
+        const resetResponse = questionIndex => {
+            const allResponseIds = Object.keys(itemsApp.getQuestions());
+            const currentResponseId = allResponseIds[questionIndex];
+
+            itemsApp.question(currentResponseId).resetResponse();
```

Comment:
This works fine since the keys from `Object.keys()` returns the questions in the correct order.

Alternatively, I can change it so that we filter the responses to get the correct reference using the button question index data attribute.